site_root/input/resources/examples/Specimen-bs-001.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Can you make this example as complete as possible? What I mean is, let's try to have all of our examples fill out as many of the resource attributes as possible so people can see how to use them. For example, I see this is missing `collection.method`, `collection.bodySite`, `receiveTime` etc.

Reply:
No problem. Per `age_at_event_days`, we can import your `age-at-event` extension once the phenotype PR is merged in.

Per `analyte_type`, we discussed that this should be mapped into a new `sequencing_experiment` profile using FHIR MolecularSequence, so the unnecessary attributes to the essence of a specimen such as  `analyte_type` and `sequencing_center_id` can be separated. That's why this wasn't included.

Per `consent_type`, we also wanted to discuss whether it should be at a specimen-level or we'd want to use FHIR Consent, instead.

I will create two tickets about these.